src/transformers/models/mixtral/modeling_mixtral.py

```diff
@@ -1467,6 +1467,7 @@ def prepare_inputs_for_generation(
                 "past_key_values": past_key_values,
                 "use_cache": kwargs.get("use_cache"),
                 "attention_mask": attention_mask,
+                "output_router_logits": False,
```

Comment:
```suggestion
                "output_router_logits": output_router_logits,
```
let's add `output_router_logits` to the list of inputs to this function and default it to `False` 😉